cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -382,7 +383,16 @@ mkSyncEnv ::
   IO SyncEnv
 mkSyncEnv trce backend connectionString syncOptions protoInfo nw nwMagic systemStart syncNodeConfigFromFile syncNP ranMigrations runMigrationFnc = do
   dbCNamesVar <- newTVarIO =<< dbConstraintNamesExists backend
-  cache <- if soptCache syncOptions then newEmptyCache 250000 50000 else pure useNoCache
+  cache <-
+    if soptCache syncOptions
+      then
+        newEmptyCache
+          LRUCacheCapacity
+            { lirCapacityStakeHashRaw = 1500000
```

Comment:
From a recent full sync to mainnet
```
[db-sync-node:Info:79] [2024-06-19 15:03:48.27 UTC] 
Cache Statistics:
  Stake Addresses: cache size: 1664030, hit rate: 89%, hits: 656453661, misses: 73037617
  Pools: cache size: 5878, hit rate: 99%, hits: 536552899, misses: 472490
  Datums: cache capacity: 50000, cache size: 50000, hit rate: 42%, hits: 19769700, misses: 26260235
  Multi Assets: cache capacity: 250000, cache size: 250000, hit rate: 75%, hits: 715052104, misses: 234336378
  Previous Block: hit rate: 49%, hits: 9554635, misses: 9554637
[db-sync-node:Info:79] [2024-06-19 15:03:48.27 UTC] Starting epoch 492
```
I would say we should increase it to 2000000

Reply:
I'll check how much this increases our RAM usage as it might be slightly too much.

Reply:
Just been syncing with `mainnet` and epoch it's jumped back up to 40 mins, I think it's because each lookup on the LRU is more expensive due to the increase of the cache capacity